puble.css

```
@@ -0,0 +1,117 @@
+.container {
+  width: 564px;
+
+  display: flex;
+  flex-direction: column;
+  border: solid 1px #9a9a9a;
+  color: #9a9a9a;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serifZ;
+}
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 45px 16px 16px 16px;
+  border-bottom: solid 1px #c4c4c4;
+}
+.shopping-img {
+  width: 32px;
+  height: 32px;
+  cursor: pointer;
+}
+.hamburger-img {
+  width: 28.5px;
+  height: 18px;
+  cursor: pointer;
+}
+.icon {
+  width: 60px;
+  height: 60px;
+  color: white;
+  background-color: #c4c4c4;
+  border-radius: 50px;
+
+  margin: 24px 24px 0px 482px;
+  text-align: center;
+  border: none;
+  cursor: pointer;
+}
+.icon:hover {
+  transform: scale(1.1);
+  color: red;
+}
+.icon-heart {
+  cursor: pointer;
+  font-size: 24px;
+}
+.icon-number {
+  font-size: 14px;
+}
+.text {
+  text-align: center;
+  border-bottom: solid 1px #c4c4c4;
+}
+.제목 {
+  color: #606060;
+}
+.제목글 {
+  color: #a1a1a1;
+  font-size: 16px;
+}
+.money {
+  font-size: 24px;
+  color: rgba(255, 143, 142);
+}
+
+.menu {
+  display: flex;
+  flex-direction: row;
+}
+label {
+  cursor: pointer;
+}
+.menu {
+  display: flex;
+  justify-content: space-around;
+  padding: 20px;
+  border-bottom: solid 1px #c4c4c4;
+}
+.radio {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.radio :hover {
+  transform: scale(1.1);
+}
+.img-coffee {
+  width: 101px;
+  height: 103px;
+}
+.location {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 28px;
+  border-bottom: solid 1px #c4c4c4;
+  align-items: center;
+}
+.location-text + .location-change {
```

어떤 것을 의도한 코드인지는 모르겠으나 다음과 같이 A + B 형태로 선택자를 사용할 경우 A와 B 둘 다 선택하는 것이 아니라 A 바로 뒤에 있는 형제 B를 선택하게 됩니다. A와 B 모두 선택하고 싶었다면 + 대신 ,를 사용하세요.
```suggestion
.location-text, .location-change {
```

또, 클래스명의 시작이 location- 으로 겹치기 때문에 중첩 선택자를 사용하면 더 간편해집니다.
```suggestion
[class^='location-'] {
```